tests/kafkatest/services/verifiable_producer.py

```diff
@@ -209,24 +182,16 @@ def start_cmd(self, node, idx):
             cmd += " --acks %s " % str(self.acks)
 
         cmd += " --producer.config %s" % VerifiableProducer.CONFIG_FILE
-        cmd += " 2>> %s 1>> %s &" % (VerifiableProducer.STDERR_CAPTURE, VerifiableProducer.STDOUT_CAPTURE)
+        cmd += " 2>> %s | tee -a %s &" % (VerifiableProducer.STDOUT_CAPTURE, VerifiableProducer.STDOUT_CAPTURE)
```

Comment:
This seems wrong, we have 2 `STDOUT_CAPTURE` and no `STDERR_CAPTURE`. Why did the first one change to `STDOUT` since we're still redirecting stderr to the file?

(It's also not obvious to me why we changed this to use `tee` but it seems to not affect anything critical here.)